src/NovelLoggingService.h

```diff
@@ -0,0 +1,62 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT License (MIT). See LICENCE.md in the repository root for more information.
+
+#ifndef NOVELRT_NOVELLOGGINGSERVICE_H
+#define NOVELRT_NOVELLOGGINGSERVICE_H
+
+#include <iostream>
+#include "spdlog/spdlog.h"
+#include "spdlog/sinks/stdout_color_sinks.h"
+
+namespace NovelRT {
```

Comment:
Same here, do we want to adhere to the new formatting rules weve been discussing in Discord since this is a new type?

Reply:
Yep - indented types that come after the namespace for NovelLoggingService.